src/bot/bot.py

```
@@ -109,21 +100,26 @@ def parse_datetime(date_str: str, time_str: str) -> datetime:
+# Replace TOKEN and APP_ID assignment
+TOKEN = config.DISCORD_TOKEN
+APP_ID = config.APP_ID
+
+def validate_user_id(user_id: str) -> int:
+    """Validate and convert user ID to integer"""
+    if not user_id:
+        raise ValueError("Owner user ID not set in environment variables")
+    try:
+        return int(user_id)
+    except ValueError:
+        raise ValueError("Invalid owner user ID. Must be an integer")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)